error-handlers.js

```diff
@@ -1,14 +1,14 @@
 exports.psqlErrorHandler = (err, req, res, next) => {
   if (err.code === "22P02") {
-    console.log("err 22P02");
+    res.status(400).send({ msg: "Invalid article ID" });
   } else {
     next(err);
   }
 };
 
 exports.customErrorHandler = (err, req, res, next) => {
-  if (err.status && err.message) {
-    console.log(err.status);
+  if (err.status && err.msg) {
+    res.status(404).send({ msg: "ID not found" });
```

Comment:
You are not really using the custom error being thrown here, even though you are checking if the status and msg properties exist on the error, you are still hardcoding the status and msg in your response.